code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -168,6 +160,15 @@ var/const/MAX_ACTIVE_TIME = 400
 									"<span class='userdanger'>[src] smashes against [H]'s [H.head]!</span>")
 				Die()
 				return FALSE
+
+		if(target.wear_mask)
+			var/obj/item/clothing/W = target.wear_mask
+			if(W.flags & NODROP)
+				return FALSE
+			if(!istype(W,/obj/item/clothing/mask/facehugger))
+				target.dropItemToGround(W)

```

Comment:
dropItemToGround already checks for the NODROP flag, you can just do `if(!istype(W ,..) && target.dropItemToGround(W)`